fvtest/tril/test/ParserTest.cpp

```diff
@@ -128,29 +127,35 @@ TEST(ParserTest, SingleNodeWithHexArg) {
     auto arg = trees->getArgs();
     ASSERT_STREQ("", arg->getName());
     ASSERT_EQ(ASTValue::Integer, arg->getValue()->getType());
-    ASSERT_EQ(0xabc123, arg->getValue()->getInteger());
+    ASSERT_EQ(GetParam().second, arg->getValue()->getInteger());
     ASSERT_NULL(arg->next);
 }
 
-TEST(ParserTest, SingleNodeWithNegativeHexArg) {
-    auto trees = parseString("(nodeName -0x249DEF)");
+INSTANTIATE_TEST_CASE_P(
+    ParserTest,
```

Comment:
For consistency with the other test, I would rather this be `ParserTestSingleNodeWithIntArgAsDecValue`.